### DISCUSSION TITLE
404 on release assets

### DISCUSSION THREAD

**Discussion by mmebsout** [community/community#44759] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

Hello, I tried to download release assets like : 

https://github.com/NASA-PDS/validate/releases/download/v3.1.1/validate-3.1.1-bin.tar.gz 

but got a 404. 

I also tried 
https://github.com/astropy/astropy/archive/refs/tags/v5.2.1.zip 
and got a 404 as well. 

can someone confirm that there is a problem with github ?

**Comment by mmebsout** [comment, marked as answer]
oh well I could get them finally with curl -L  or using Chrome but had to upgrade Firefox for it to work there.

**Comment by iiotdatareader** [comment]
It's still not working for me with -L option. Even passed access token header but always gives 404 for the URL copied on asset.
